src/ng5-slider/lib/slider.component.ts

```diff
@@ -1157,22 +1161,28 @@ export class SliderComponent implements OnInit, AfterViewInit, OnDestroy, Contro
 
   // Update position of the floor label
   private updateFloorLab(): void {
-    this.setLabelValue(this.getDisplayValue(this.minValue, LabelType.Floor), this.flrLabElem);
-    this.calculateElementDimension(this.flrLabElem);
-    const position: number = this.viewOptions.rightToLeft
-      ? this.barDimension - this.flrLabElem.dimension
-      : 0;
-    this.setPosition(this.flrLabElem, position);
+    if (!this.flrLabElem.alwaysHide) {
+      this.setLabelValue(this.getDisplayValue(this.minValue, LabelType.Floor), this.flrLabElem);
+      let position: number = 0;
+      if (this.viewOptions.rightToLeft) {
+        this.calculateElementDimension(this.flrLabElem);
```

Comment:
I would call `this.calculateElementDimension(this.flrLabElem)` regardless of `rightToLeft` option, because there are other places in code which access `this.flrLabElem.dimension` and may depend on it getting updated here.

Reply:
I updated the pull request to include the requested calls to `this.calculateElementDimension()`